# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libixp/build.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libixp-0.5/config.mk.vanilla	2008-09-23 16:53:47.000000000 +0200
+++ libixp-0.5/config.mk	2008-09-23 16:54:19.000000000 +0200
@@ -22,7 +22,8 @@
 LDFLAGS = -g $(LDLIBS) $(LIBS)
 
 # Compiler, Linker. Linker should usually *not* be ld.
-CC = cc -c
+CC = cc
+override CC += -c
 LD = cc
 # Archiver
 AR = ar crs
